tests/artifacts/test_artifacts.py

```diff
@@ -67,3 +67,69 @@ def test_download_artifacts_throws_for_invalid_arguments():
 
     with pytest.raises(MlflowException, match="`artifact_path` cannot be specified"):
         mlflow.artifacts.download_artifacts(artifact_path="path", artifact_uri="uri")
+
+
+@pytest.fixture()
+def run_with_text_artifact():
+    artifact_path = "test/file.txt"
+    artifact_content = "This is a sentence"
+    with mlflow.start_run() as run:
+        mlflow.log_text(artifact_content, artifact_path)
+
+    artifact_uri = str(pathlib.PurePosixPath(run.info.artifact_uri) / artifact_path)
+    return (artifact_uri, run, artifact_path, artifact_content)
```

Comment:
Can we use `namedtuple` instead of plain `tuple`?

Reply:
Example:

```python
Artifact = namedtuple("Artifact", ["uri", "content"])

@pytest.fixture()
def run_with_text_artifact():
    artifact_path = "test/file.txt"
    artifact_content = "This is a sentence"
    with mlflow.start_run() as run:
        mlflow.log_text(artifact_content, artifact_path)

    artifact_uri = str(pathlib.PurePosixPath(run.info.artifact_uri) / artifact_path)
    return Artifact(artifact_uri, content)


def test_load_text(run_with_text_artifact):
    artifact = run_with_text_artifact
    assert mlflow.artifacts.load_text(artifact.uri) == artifact.content
```

Reply:
Great suggestion !!